### ISSUE TITLE
better detection of file arguments

### ISSUE THREAD

**Issue by sbidoul** [acsone/kwkhtmltopdf#4]
Currently, the client detects file arguments with a simple heuristics: an argument that does not start with `-`  and corresponds to an existing file is considered a file argument.

This may break in rare situations where an existing file correspond to an argument name.

**Comment by foutoucour** [comment]
Hey @sbidoul did you consider [cobra](https://github.com/spf13/cobra) to manage the arguments ?
It seems a good tool. I used it for some cli here at Numigi. It was super easy.

**Comment by sbidoul** [comment]
Nope. Not sure it would help though. The problem is that we don't know (and don't want to reproduce) the exact cli of wkhtmltopdf. So I don't really know how to fix this bug, but it's probably not an issue in practice.
